30-helm/multi-region/pingfederate/base.yaml

```diff
@@ -0,0 +1,69 @@
+global:
+  ingress:
+    enabled: true
+    addReleaseNameToHost: prepend
+    defaultDomain: ping-devops.com
+    defaultTlsSecret:
+    annotations:
+      nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"
+      kubernetes.io/ingress.class: "nginx-public"
+
+pingfederate-admin:
+  enabled: true
+  container:
+    replicaCount: 1
+  envs:
+    SERVER_PROFILE_BASELINE_URL: https://github.com/pingidentity/pingidentity-server-profiles.git
+    SERVER_PROFILE_BASELINE_PATH: baseline/pingfederate
+    SERVER_PROFILE_PARENT: BASELINE
+    ## Using this profile layer for PF_NODE_GROUP_ID var
```

Comment:
If it's not harmful, we should just add that file to baseline methinks. if we can get rid of a profile, great.

Reply:
Good suggestion. I looked at the PF default and it seems like it's no issue to add to baseline, so I'll do that too

Reply:
https://github.com/pingidentity/pingidentity-server-profiles/pull/109